packages/server/src/cli/main.rs

```diff
@@ -70,9 +70,9 @@ fn select_environment() -> Result<usize, Box<dyn std::error::Error + Send + Sync
 fn select_action() -> Result<usize, Box<dyn std::error::Error + Send + Sync>> {
     let selections = &[
         "Initialize new E3 round.",
-        "Activate an E3 round.",
         "Participate in an E3 round.",
-        "Decrypt Ciphertext & Publish Results",
+        // "Activate an E3 round.",
+        // "Decrypt Ciphertext & Publish Results",
```

Comment:
Noted from discord that these are being removed because they are redundant.

Reply:
yes, they were used previously as a test for contracts as @hmzakhalid pointed out. I left the code but hide from the user perspective